### PR TITLE
Allow configuration of session tracking mode

### PR DESCRIPTION
**Pull request by roysjosh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -863,6 +863,9 @@ List of request URI mapped to the `jsp` servlet. Defaults to `['*.jsp', '*.jspx'
 #####`sessionconfig_sessiontimeout`
 Default session timeout for applications, in minutes. Defaults to `30`.
 
+#####`sessionconfig_trackingmode`
+Default session tracking mode for applications. Defaults to `URL`, `COOKIE` (unless the context parameter cookies=false) and `SSL` (if a connector used by this context has secure=true).
```
Comment (roysjosh) [inline README.md]: This logic is in tomcat itself.  Should I keep this language?

Comment (antoineco) [reply README.md]: Yes it looks clear to me, could you just reference some relevant documentation page?

**Comment by roysjosh** [comment]
In case github didn't send you an update, this PR was updated with a documentation link.  Let me know if there are any other changes you would like.

**Comment by antoineco** [comment]
Oh that's right, Github doesn't notify on "force" pushes. Thanks for the update!

**Pull request merged**